openmc/data/njoy.py

```diff
@@ -445,7 +424,8 @@ def make_ace(filename, temperatures=None, acer=True, xsdir=None,
 
 def make_ace_thermal(filename, filename_thermal, temperatures=None,
                      ace='ace', xsdir=None, output_dir=None, error=0.001,
-                     iwt=2, evaluation=None, evaluation_thermal=None, **kwargs):
+                     iwt=2, evaluation=None, evaluation_thermal=None,
+                     table_name=None, zaids=None, nmix=None, **kwargs):
```

Comment:
One thing that isn't clear to me is the mode of failure if a `table_name`, `zaids`, `nmix` combination is provided to the which isn't present in the thermal data file. We would get a failure from `njoy` I suppose?

Reply:
Would it be possible to check that the `ThermalTuple` we're using is valid for the filenames passed before we get to parsing that data?

Reply:
These parameters are all things that the thermal scattering evaluation _doesn't_ tell you but that you need to input in NJOY. The normal mode of operation is that we infer them from the ZSYMAM value, but in the event that a user is processing some new file that we haven't accounted for yet, these parameters give them a way to manually specify the needed parameters.